File: Szhchem.Data/Models/Description.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Szhchem.Data
{
    /// <summary>
    /// 多语言描述文字
    /// </summary>
    public class Description
    {
        public int LanguageCode { get; set; }
        public string Title { get; set; }
        public string DetailText { get; set; }
        public string Url { get; set; }
    }
}

File: molfinder/ClientApp/src/models/Description.ts
export default class Description {
    public languageCode: number;
    public title: string;
    public detailText: string;
    public url: string;

    constructor(languageCode: number, title: string, detailText: string, url: string) {
        this.languageCode = languageCode;
        this.title = title;
        this.detailText = detailText;
        this.url = url;
    }
}